frontend_lib/src/component/Tags/Tag.jsx

```diff
@@ -1,41 +1,47 @@
 import React from 'react'
 import { translate } from 'react-i18next'
-import { Checkbox } from '../Input/Checkbox.jsx'
 import PropTypes from 'prop-types'
+import IconButton from '../Button/IconButton.jsx'
 
 require('./TagList.styl')
 
 export const Tag = props => {
   return (
-    <div className='tagList__list__item'>
-      <Checkbox
-        onClickCheckbox={props.onClickCheckbox}
-        checked={props.checked}
-        name={'tag-' + props.tagId}
-        styleLabel={{ marginLeft: '5px', marginRight: '10px' }}
-        styleCheck={{ top: '-5px' }}
-      />
-      <label
-        htmlFor={'checkbox-tag-' + props.tagId}
-        className='tagList__list__item__info'
-        title={props.name}
-      >
-        {props.name}
-      </label>
+    <div className='tagList__list__item__wrapper'>
+      <i className='fas fa-play tagList__list__item__triangle'></i>
+      <div className='tagList__list__item'>
+        <div className='tagList__list__item__circle' />
+        <label
+          className='tagList__list__item__info'
+          title={props.name}
+        >
```

Comment:
This label is missing a `htmlFor` attribute. Turn it into span if having it as a label is not relevant anymore.